codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeInterceptorSpec.java

```diff
@@ -146,20 +147,23 @@ private MethodSpec generateAuthSchemeParams() {
                                                .addParameter(SdkRequest.class, "request")
                                                .addParameter(ExecutionAttributes.class, "executionAttributes");
 
-        if (!authSchemeSpecUtils.useEndpointBasedAuthProvider()) {
+        if (!authSchemeSpecUtils.useEndpointParamsInAuthScheme()) {
             builder.addStatement("$T operation = executionAttributes.getAttribute($T.OPERATION_NAME)", String.class,
                                  SdkExecutionAttribute.class);
-            builder.addStatement("$T.Builder builder = $T.builder().operation(operation)",
-                                 authSchemeSpecUtils.parametersInterfaceName(),
-                                 authSchemeSpecUtils.parametersInterfaceName());
-
             if (authSchemeSpecUtils.usesSigV4()) {
                 builder.addStatement("$T region = executionAttributes.getAttribute($T.AWS_REGION)", Region.class,
                                      AwsExecutionAttribute.class);
-                builder.addStatement("builder.region(region)");
+                builder.addStatement("return $T.builder()"
```

Comment:
Reverted this fix back to master , since these changes were not required as we are using same code as EndpointBased params